calibration/PCO2WB/CGINS-PCO2WB-C0051__20120828.csv

```diff
@@ -5,7 +5,7 @@ C0051,CC_calc,-2.1138,Instrument specific calibration coefficient for the pCO2 m
 C0051,CC_calt,14.2,Instrument specific calibration coefficient for temperature
 C0051,CC_ea434,19706,Parameters per vendor via Chris Wingard; Reagent specific calibration coefficient
 C0051,CC_ea620,34,Parameters per vendor via Chris Wingard; Reagent specific calibration coefficient
```

Comment:
let's remove every notes column comment on this one, thanks!

Reply:
all comments removed from notes column of this file